jib-core/src/main/java/com/google/cloud/tools/jib/image/json/ContainerConfigurationTemplate.java

```diff
@@ -124,8 +123,12 @@ public void setContainerCmd(List<String> cmd) {
     config.Cmd = cmd;
   }
 
-  public void setContainerExposedPorts(SortedMap<String, EmptyStruct> exposedPorts) {
-    config.ExposedPorts = exposedPorts;
+  public void setContainerExposedPorts(List<String> exposedPorts) {
+    SortedMap<String, Map<?, ?>> result = new TreeMap<>();
+    for (String port : exposedPorts) {
+      result.put(port + "/tcp", Collections.emptyMap());
```

Comment:
I believe we can just do port without `/tcp` since by default it is tcp.

https://github.com/opencontainers/image-spec/blob/master/config.md#properties